samples/led/main.c

```diff
@@ -1,19 +1,12 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
+    // Set front LEDs to shine red for 2*X ms every 4*X ms
+    XSetManualLED(LED_RED, LED_RED, LED_OFF, LED_OFF);
```

Comment:
Let's make this `LED_GREEN, LED_RED, LED_OFF, LED_OFF` so we can see the order (green happens before red)?